### PR TITLE
[WIP ][VCX] Getting started guide

### PR DESCRIPTION
**Pull request by dkulic**
Signed-off-by: Darko Kulic <darko.kulic@evernym.com>

### CODE REVIEW

File: vcx/docs/getting-started/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,646 @@
+﻿# Getting Started with Libindy
```
Comment (vimmerru) [inline vcx/docs/getting-started/getting-started.md]: I believe we should change the title to reference VCX

File: vcx/docs/getting-started/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+
+## A Developer Guide for Building Indy Clients Using Libindy
+
+![logo](https://raw.githubusercontent.com/hyperledger/indy-node/master/collateral/logos/indy-logo.png)
+
+* [Getting Started with Libvcx](#getting-started-with-libvcx)
+  * [What Indy and Libindy are and Why They Matter](#what-indy-and-libindy-are-and-why-they-matter)
```
Comment (vimmerru) [inline vcx/docs/getting-started/getting-started.md]: > What Indy and Libindy are and Why They Matter

Can we re-name it to What Indy, Libindy, Libvcx are and Why They Matter

File: vcx/docs/getting-started/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,646 @@
...
+      * [Step 5: Credential Schemas Setup](#step-5-credential-schemas-setup)
+      * [Step 6: Credential Definition Setup](#step-6-credential-definition-setup)
+  * [Alice Gets a Transcript](#alice-gets-a-transcript)
+  * [Apply for a Job](#apply-for-a-job)
+  * [Apply for a Loan](#apply-for-a-loan)
+  * [Explore the Code](#explore-the-code)
+
+## What Indy, Libindy and Libvcx are and Why They Matter
+
+Indy provides a software ecosystem for private, secure, and powerful identity, and libindy enables clients for it. Indy puts people — not the organizations that traditionally centralize identity — in charge of decisions about their own privacy and disclosure. Libvcx is library on top of libindy which simplifies credential exchange. This enables all kinds of rich innovation: connection contracts, revocation, novel payment workflows, asset and document management features, creative forms of escrow, curated reputation, integrations with other cool technologies, and so on.
```
Comment (vimmerru) [inline vcx/docs/getting-started/getting-started.md]: We can add sentence that libindy is a low level library that provides fine configuration, but vcx is more high-level library concentrated on hiding low-level details and increasing of application development efficiency.

File: vcx/docs/getting-started/getting-started.md (hunk 1 above)
Comment (dkulic) [reply vcx/docs/getting-started/getting-started.md]: Yes :)

File: vcx/docs/getting-started/getting-started.md (hunk 2 above)
Comment (dkulic) [reply vcx/docs/getting-started/getting-started.md]: Sure, missed this.

File: vcx/docs/getting-started/getting-started.md (hunk 3 above)
Comment (dkulic) [reply vcx/docs/getting-started/getting-started.md]: Done

**Pull request merged**